core/src/main/java/com/graphhopper/storage/BaseGraph.java

```diff
@@ -175,51 +172,45 @@ void checkInitialized() {
             throw new IllegalStateException("The graph has not yet been initialized.");
     }
 
-    protected int loadNodesHeader() {
-        nodeEntryBytes = nodes.getHeader(1 * 4);
-        nodeCount = nodes.getHeader(2 * 4);
```

Comment:
Don't know why we start the header at `1*4` not `0*4`, so I changed this, and it seems to work.